Simplify installation of udev rules (#2264)
pwr-Solaar/Solaar: docs/installation.md

### Installing Solaar's udev rule

Solaar needs to write to HID devices and receivers.

> To achieve this without Solaar running as root, requires a udev rule. Run `make install_udev` in Solaar's root folder to

Reviewer: ```suggestion
To achieve this without Solaar running as root, which is not recommended, requires a udev rule. Run `make install_udev` in Solaar's root folder to
```

> install and reload udev rule.

Reviewer: ```suggestion
install Solaar's regular udev rule and put it into effect.  You need sudo privileges to do this and will be asked for your password.  If you are using Wayland instead of X11 you may want to instead `make install_udev_uinput` so that Solaar rules can simulate input in Wayland.
```

<details>
  <summary>Manual steps</summary>

  You can install this rule manually by copying the file
  [`rules.d/42-logitech-unify-permissions.rules`](/rules.d/42-logitech-unify-permissions.rules) as root from Solaar repo
  to `/etc/udev/rules.d`.
  Let udev reload its rules by running
  `sudo udevadm control --reload-rules`.
</details>

> Then disconnect your receivers and any USB- or Bluetooth-connected devices for the udev rule to take effect and

Reviewer: ```suggestion
Then disconnect your Logitech receivers and any USB- or Bluetooth-connected Logitech devices and
```

> re-connect them or just reboot your computer.

Reviewer: ```suggestion
re-connect them for the udev rule to take effect on them.  Alternatively, you can just reboot your computer.
```
